R/covidcast.R

```diff
@@ -32,10 +32,10 @@ parse_signal <- function(signal, base_url) {
 }
 
 #' @export
-print.covidcast_data_signal <- function(signal, ...) {
```

Comment:
Not sure why you're preferring a more generic variable name?

Reply:
We're required to use `x`, because that's how `print()` is defined in base R:

```
> print
function (x, ...) 
UseMethod("print")
```

See https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Generic-functions-and-methods